main.go

```diff
@@ -737,6 +739,41 @@ func (u *User) changeRoom(r *Room) {
 	u.room.broadcast("", Green.Paint(" --> ")+u.Name+" has joined "+Blue.Paint(u.room.name))
 }
 
+func (u *User) showPrompt() {
```

Comment:
is there a way to only go through the whole prompt generating function when the prompt changes? right now it generates your prompt string from scratch every single time

Reply:
That's a good idea. I will try to implement it.

Reply:
Done!